src/snap/actions/pairing.ts

```diff
@@ -134,7 +140,16 @@ export const getToken = async (currentAccountAddress?: string) => {
 		if (currentAccountAddress && tempAccountAddress == null) {
 			await updatePairingObject(data.token, pairingDataInit.pairingId, {
 				isPaired: false,
-				pairingFailureReason: 'NO_BACKUP_DATA_WHILE_REPAIRING',
+				pairingRemark: PairingRemark.NO_BACKUP_DATA_WHILE_REPAIRING,
+			});
+		} else if (
+			currentAccountAddress &&
+			tempAccountAddress &&
+			currentAccountAddress !== tempAccountAddress
+		) {
+			await updatePairingObject(data.token, pairingDataInit.pairingId, {
```

Comment:
`updatePairingObject` doesn't give a full context of what the function is doing. Should we consider a name that can give more context? `updatePairingDocFirestore`